modules/standard/Set.chpl

```diff
@@ -234,17 +234,28 @@ module Set {
 
       on this {
         _enter(); defer _leave();
-        var (hasFoundSlot, _) = _htb.findFullSlot(x);
-        result = hasFoundSlot;
+        result = _contains(x);
       }
 
       return result;
     }
 
+    /*
+     As above, but parSafe lock must be held and must be called "on this".
+    */
+    proc const _contains(const ref x: eltType): bool {
```

Comment:
Can we get a `pragma "no doc"` on this helper method?

Reply:
Oops, sorry about that.